test/Erc20Quest.spec.ts

```diff
@@ -335,32 +335,36 @@ describe('Erc20Quest', async () => {
     })
 
     it('should transfer non-claimable rewards back to owner', async () => {
-      await deployedFactoryContract.connect(firstAddress).mintReceipt(questId, messageHash, signature)
+      await deployedRabbitholeReceiptContract.setMinterAddress(deployedFactoryContract.address)
+
+      await deployedFactoryContract.mintReceipt(questId, messageHash, signature)
       await deployedQuestContract.start()
       await ethers.provider.send('evm_increaseTime', [expiryDate + 100])
-      await deployedQuestContract.connect(firstAddress).claim()
-
+      await deployedQuestContract.connect(owner).claim()
       await ethers.provider.send('evm_increaseTime', [expiryDate + 1000])
       await deployedQuestContract.withdrawRemainingTokens(owner.address)
 
       expect(await deployedSampleErc20Contract.balanceOf(deployedQuestContract.address)).to.equal(200)
       expect(await deployedSampleErc20Contract.balanceOf(owner.address)).to.equal(
-        totalRewardsPlusFee * 100 - 1 * 1000 - 200
+        // totalRewardsPlusFee * 100 - 1 * 1000 - 200
```

Comment:
I had to change the math here. @GarrettJMU or @jonathandiep is this correct? Not sure if it was incorrect before or now.